webview/src/experiments/components/table/styles.module.scss

```diff
@@ -774,18 +628,108 @@ td {
   font-size: 0.65rem;
 }
 
-.error {
-  color: $error-color;
+.bullet {
+  visibility: visible;
+  align-items: center;
+  background: $checkbox-background;
+  border-radius: $bullet-size;
+  border: calc(var(--border-width) * 1px) solid $checkbox-border;
+  color: $icon-color;
+  cursor: pointer;
+  display: flex;
+  height: $bullet-size;
+  justify-content: center;
+  outline: none;
+  padding: 0;
+  position: relative;
+  width: $bullet-size;
+
+  &::before {
+    display: inline-block;
+    position: relative;
+    content: '';
+    border-radius: 99px;
+    width: 6px;
+    height: 6px;
+    z-index: 2;
+
+    .normalExperiment & {
+      line-height: 0;
+      background: currentColor;
+      border-radius: 100%;
+    }
+
+    .unselectedExperiment & {
+      width: 4px;
+      height: 4px;
+      vertical-align: middle;
+      border: 1px solid $icon-color;
+      background-color: $checkbox-background;
+    }
+
+    .runningExperiment & {
+      width: 4px;
+      height: 4px;
+      vertical-align: middle;
+      border: 1.5px solid $checkbox-background;
+      border-radius: 100%;
+      border-right: 1.5px solid currentColor;
+      border-top: 1.5px solid currentColor;
+      animation: spin 1s cubic-bezier(0.53, 0.21, 0.29, 0.67) infinite;
+      background-color: $checkbox-background;
+    }
+  }
 }
 
-.experimentCellContentsContainer {
-  @extend .cellContents;
```

Comment:
Personally, I don't think it's the best practice to extend regular classes because someone could update the class not realizing they're effecting other classes. Created another one of our "Extendable Silent Rules" for this.